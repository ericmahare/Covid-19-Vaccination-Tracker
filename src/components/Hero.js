import React from 'react'
const Hero = ({ map, country, vaccinated, total}) => {
  return (
    <div className='hero'>
      <div className='hero-img-container'>
        <img src={map} alt='Africa' className={country ? 'flag' : 'hero-img'}/>
      </div>
      <div className='hero-info'>
        <h2>{country ? country : 'Africa Total Vaccination'}</h2>
        <h2>{vaccinated ? `(${vaccinated.toLocaleString()}) people vaccinated` : `${total.toLocaleString()} people vaccinated` }</h2>
      </div>
    </div>
  )
}

export default Hero