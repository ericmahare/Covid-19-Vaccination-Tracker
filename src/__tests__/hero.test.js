import renderer from 'react-test-renderer';
import Hero from '../components/Hero';

const map = './images/africa.svg';
const country = 'Kenya';
const vaccinated = 40000;
const total = 100000;

it('renders correctly', () => {
  const tree = renderer.create(
    <Hero map={map} country={country} vaccinated={vaccinated} total={total}/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
})