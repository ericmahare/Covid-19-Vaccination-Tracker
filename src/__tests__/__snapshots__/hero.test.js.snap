// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="hero"
>
  <div
    className="hero-img-container"
  >
    <img
      alt="Africa"
      className="flag"
      src="./images/africa.svg"
    />
  </div>
  <div
    className="hero-info"
  >
    <h2>
      Kenya
    </h2>
    <h2>
      (40,000) people vaccinated
    </h2>
  </div>
</div>
`;
